### PR TITLE
Feature: Instances operator commands

### PR DESCRIPTION
**Pull request by 1yam**
**Note:** This pull request depends on the following PR and should not be merged before it:
[https://github.com/aleph-im/aleph-sdk-python/pull/124](https://github.com/aleph-im/aleph-sdk-python/pull/124)

#### Summary
This PR introduces new commands for managing virtual machine instances. These commands provide users with enhanced control over their VMs.

#### New Commands
The following commands have been added:

1. **Notify Instance**: 

   ```shell
   aleph instance notify {vm_id} {domain}
   ```

   Sends a notification to the specified VM instance within the given domain.

2. **Stop Instance**:

   ```shell
   aleph instance stop {vm_id} {domain}
   ```

   Stops the specified VM instance within the given domain.

3. **Reboot Instance**:

   ```shell
   aleph instance reboot {vm_id} {domain}
   ```

   Reboots the specified VM instance within the given domain.

4. **Erase Instance**:

   ```shell
   aleph instance erase {vm_id} {domain}
   ```

   Erases the specified VM instance within the given domain, securely deleting all data.

5. **Expire Instance**:

   ```shell
   aleph instance expire {vm_id} {domain}
   ```

### CODE REVIEW

**Comment by Psycojoker** [comment]
@1yam when you have some time, can you rebase this PR? It conflics with master

File: setup.cfg [hunk 1]
```diff
@@ -31,7 +31,7 @@ package_dir =
 setup_requires = pyscaffold>=3.2a0,<3.3a0
 # Add here dependencies of your project (semicolon/line-separated), e.g.
 install_requires =
-    aleph-sdk-python~=0.9.1
+    aleph-sdk-python@git+https://github.com/aleph-im/aleph-sdk-python.git@1yam-vm-client#egg=aleph-sdk-python
```
Comment (Psycojoker) [inline setup.cfg:34]: TODO: remove this once the SDK has been released

**Comment by hoh** [comment]
Not sure what happened, but the history of this branch seems messed up (some commits from main reappear in the history), and I see many conflicts. What is the plan regarding that ? Squash everything or cleanup ?

**Comment by 1yam** [comment]
Gonna fix conflicts and gonna squash those commit

**Comment by 1yam** [comment]
well was more simple to just re open new one: [PR](https://github.com/aleph-im/aleph-client/pull/225)

**Pull request closed without merging**